### ISSUE TITLE
Function Request: Storage Exposer

### ISSUE THREAD

**Issue by yuanju-qwq** [GTNewHorizons/Applied-Energistics-2-Unofficial#547]
Can you port the storage exposer from the 1.12.2 version of nae2mod?
This would help subnets have access to specific items on the mainnet, and would make it easier to overhaul subnet spawn lines!

**Comment by MCTBL** [comment]
could u explain what can this `storage exposer` do and how it works?

**Comment by StaffiX** [comment]
[There's the Github Wiki Page for it](https://github.com/AE2-UEL/NAE2/wiki/ME-Storage-Exposer) but considering storage bus filtering on import mode doesn't work, I'm not sure how this would help

**Comment by querns** [comment]
> [There's the Github Wiki Page for it](https://github.com/AE2-UEL/NAE2/wiki/ME-Storage-Exposer) but considering storage bus filtering on import mode doesn't work, I'm not sure how this would help

Based on what I read in that page, this block allow you to roughly do the same thing as a filterable extract-only storage bus. I would very much like to have either this or a filterable extract-only storage bus. Right now, if you want to expose just part of a network's storage, you have to use chest buffers and fluid tanks with the large network constantly exporting items to the intermediate storage. It's a bunch of boilerplate for every setup.

**Comment by StaffiX** [comment]
I dont think this would work without fixing storage bus import mode, because AE2UEL (the 1.12.2 ae2 fork) fixed that as well

**Comment by EnderProyects** [comment]
It could also be a nice QoL the port of these two from NAE2
https://github.com/AE2-UEL/NAE2/wiki/ME-Beam-Former
https://github.com/AE2-UEL/NAE2/wiki/Job-Auto-Complete-Card
